flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/TestFlinkTableSink.java

```diff
@@ -60,10 +63,23 @@ public static List<Object[]> parameters() {
         for (Object[] catalogParams : CatalogTestBase.parameters()) {
           String catalogName = (String) catalogParams[0];
           Namespace baseNamespace = (Namespace) catalogParams[1];
-          parameters.add(new Object[] {catalogName, baseNamespace, format, isStreaming});
+          boolean doNotUseV2Sink = false;
```

Comment:
we have a kind of double negation `doNotUseV2Sink = false` , but below in the other block we have `boolean useV2Sink = true;`
could we just do
```java
parameters.add(
              new Object[] {catalogName, baseNamespace, format, isStreaming, false});
```
ie: removing this local variable?

Reply:
In [Contribute guide](https://iceberg.apache.org/contribute/?h=contri#boolean-arguments) is written that boolean arguments should be named. This rule shouldn't be applied in this place as well? Or do you prefer the comment in stead?

Reply:
I replaced it with comment, take a look now